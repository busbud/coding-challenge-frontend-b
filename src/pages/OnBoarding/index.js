import React from 'react';
import Grid from 'material-ui/Grid';
import Collapse from 'material-ui/transitions/Collapse';
import Jumbotron from '../../components/Jumbotron';
import SearchForm from '../../containers/SearchForm';
import SearchResults from '../../containers/SearchResults';

export default () => (
  <Grid container justify="center">
    <Grid item md={8} xs={11}>
      <Collapse in={true}>
        <Jumbotron />
      </Collapse>
      <SearchForm />
      <SearchResults />
    </Grid>
  </Grid>
);
