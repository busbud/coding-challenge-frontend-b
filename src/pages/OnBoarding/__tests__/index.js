import React from 'react';
import { shallow } from 'enzyme';
import OnBoarding from '../index';

describe('pages | OnBoarding | index', () => {
  describe('render', () => {
    it('should render without throwing and error', () => {
      // given
      const wrapper = shallow(<OnBoarding />);

      // then
      expect(wrapper).toBeDefined();
    });

    it('should match snapshot', () => {
      // given
      const wrapper = shallow(<OnBoarding />);

      // then
      expect(wrapper).toMatchSnapshot();
    });
  });
});
