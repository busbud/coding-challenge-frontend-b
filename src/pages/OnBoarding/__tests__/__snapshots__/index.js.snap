// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | OnBoarding | index render should match snapshot 1`] = `
<WithStyles(GridWrapper)
  container={true}
  justify="center"
>
  <WithStyles(GridWrapper)
    item={true}
    md={8}
    xs={11}
  >
    <WithStyles(Collapse)
      in={true}
    >
      <WithStyles(Component) />
    </WithStyles(Collapse)>
    <WithStyles(Connect(Component)) />
    <WithStyles(Connect(Component)) />
  </WithStyles(GridWrapper)>
</WithStyles(GridWrapper)>
`;
